loopchain/channel/channel_service.py

```diff
@@ -497,20 +497,90 @@ def connect_to_radio_station(self, is_reconnect=False):
                                                block_height=self.block_manager.get_blockchain().block_height)
 
     async def __subscribe_call_by_rest_stub(self, rs_rest_stub):
-        if conf.REST_SSL_TYPE == conf.SSLAuthType.none:
-            peer_target = ChannelProperty().rest_target
+        response = {'response_code': message_code.Response.fail,
+                    'message': message_code.get_response_msg(message_code.Response.fail)}
+
+        try:
+            if conf.REST_SSL_TYPE == conf.SSLAuthType.none:
+                peer_target = ChannelProperty().rest_target
+            else:
+                peer_target = f"https://{ChannelProperty().rest_target}"
+            response = rs_rest_stub.call(
```

Comment:
`rs_rest_stub.call()` could be blocked.
I recommnend replacing it with `await rs_rest_stub.call_async()`

Reply:
Updated to async call